english/Rupinderpal/rupinder.md

```diff
@@ -0,0 +1,4 @@
+Rupinderpal is Indian Hockey player 
+To know more about him click [here](https://en.wikipedia.org/wiki/Rupinder_Pal_Singh)
+[twitter profile](https://twitter.com/rupinderbob3?lang=en)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.